tfs/tools.py

```diff
@@ -18,59 +20,82 @@
 LOG = logging.getLogger(__name__)
 
 
-def significant_digits(value: float, error: float) -> (str, str):
-    """Computes value and its error properly rounded with respect to the size of the error"""
+def significant_digits(
+    value: float, error: float, return_floats: bool = False
+) -> List[Union[str, float]]:
+    """
+    Computes value and its error properly rounded with respect to the size of the error.
+
+    Args:
+        value (float): a number.
+        error (float): the error on the number.
+        return_floats (bool): if True, returns significant digits as floats, otherwise as strings.
+        Defaults to False.
+
+    Returns:
+        A tuple of the rounded value and error with regards to the size of the error.
+    """
     if error == 0:
         raise ValueError("Input error of 0. Cannot compute significant digits.")
     digits = -int(np.floor(np.log10(error)))
     if np.floor(error * 10 ** digits) == 1:
         digits = digits + 1
-    return (
+    res = [
         f"{round(value, digits):.{max(digits, 0)}f}",
         f"{round(error, digits):.{max(digits, 0)}f}",
-    )
+    ]
+    if return_floats:
+        return [float(val) for val in res]
+    return res
```

Comment:
I would not change tuple to list, the values are dependent and it does not make sense to update one without the other.